Rldx/Rldx/Managers/ResourceManager/DxResourceManager.cpp

```diff
@@ -147,28 +147,31 @@ namespace rldx
 		}*/
 	}
 
-	void DxResourceManager::GetResourcesFromCallBack(QList<QString>& qstrMissingFiles, QList<QByteArray>& destBinaries)
+	void DxResourceManager::GetResourcesFromCallBack(std::vector<std::wstring>* qstrMissingFiles, std::vector<std::vector<unsigned char>>* destBinaries)
```

Comment:
In this case is proper c++ to do
`void DGetResourcesFromCallBack(const std::vector<std::wstring>& qstrMissingFiles, std::vector<std::vector<uint8_t>>& destBinaries);`

In c++ alwayus use references if possible, and unt8_t is like the u8 i rust, it is used when you need to sure of the size

Reply:
Fixed in the last commit.